c/src/ml-api-inference-single.c

```diff
@@ -526,6 +526,7 @@ invoke_thread (void *arg)
     /* Clear input data after invoke is done. */
     ml_tensors_data_destroy (input);
     single_h->invoking = FALSE;
+    g_cond_broadcast (&single_h->cond);
```

Comment:
check line 551, move this after all value is set clearly (after line 567)

Reply:
Thank you. I agree with you. It was moved to line 567.